scipy/optimize/optimize.py

```diff
@@ -1864,7 +1864,7 @@ def get_result(self, full_output=False):
             return self.xmin
 
 
-def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
+def brent(func, args=(), bracket=None, tol=1.48e-8, full_output=0, maxiter=500):
```

Comment:
This changes the public API, so will break code for people that use the keyword. Please remove.